### DISCUSSION TITLE
2022-06-16 Meeting Agenda

### DISCUSSION THREAD

**Discussion by timbrisc** [ucum-org/ucum#191] [Meetings]
### 2022-06-16, 4:00 pm–5:00 pm EDT

**Meeting Chair: Marjorie Rallins**

Current list of units discussed and status of each, updated from previous meeting:
[2022 UCUM Units proposed_2022-05-26 Update.xlsx](https://github.com/ucum-org/ucum/files/8873030/2022.UCUM.Units.proposed_2022-05-26.Update.xlsx)

**Time**
Lead(s)
Agenda Topics

**4:00-4:10**
Marjorie Rallins
Opening remarks - process for committee governance

**4:10-4:25**
Tim Briscoe
UCUM migration to GitHub
- GitHub user names
- Migration plan
- Documentation repository
- Discussion and comment space

**4:35-4:55**
Eza Hafeza, Clem McDonald
Continue proposed units discussion
- {Zscore} – [Zscore]
- [Tscore]
- {CPM} – [CPM], counts per minute
- {Pack_years} – [Pack_years], packages of cigarettes with 20 cigarettes per pack

**4:55-5:00**
All
Other topics
- Next meeting dates

**5:00**
Marjorie Rallins
Adjourn

@ucum-org/advisors @ucum-org/regenstrief

**Comment by aflackey** [comment]
Adding the updated list of units under discussion:  [2022 UCUM Units proposed_2022-06-16.xlsx](https://github.com/ucum-org/ucum/files/8921272/2022.UCUM.Units.proposed_2022-06-16.xlsx)

**Comment by aflackey** [comment]
Did not reach a quorum - meeting canceled.
